dvc/utils/ls_colors.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+import dircolors
+
+from dvc.compat import fspath
+
+
+class LsColors(dircolors.Dircolors):
+    def __init__(self, load=True):
+        super().__init__(load)
+
+        if "dvc" not in self._codes:
+            self._codes["dvc"] = "01;31"  # Red
+
+        if ".dvc" not in self._extensions:
+            self._extensions[".dvc"] = "01;33"  # Yellow
+
+    def format(self, fs_node, text=None):
+        text = text or fspath(fs_node.path_info)
+        if not self.loaded:
+            return text
+
+        if fs_node.is_out:
+            return self._format_code(text, "dvc")
+
+        try:
+            statbuf = os.stat(fspath(fs_node.path_info))
```

Comment:
You could simply check that while doing `list` and include stat or stat-like structure in FSNode. That way your `is_out` attribute would be generalized with the rest of the stuff as well :slightly_smiling_face: Though I might be missing something here.